app/src/main/java/com/orhanobut/waspsample/WaspApplication.java

```diff
@@ -40,9 +40,11 @@ public WaspRetryPolicy getRetryPolicy() {
         service = new Wasp.Builder(this)
                 .setEndpoint("http://httpbin.org")
                 .setLogLevel(LogLevel.ALL)
-               // .setRequestInterceptor(interceptor)
+                //.setHttpStack(new OkHttpStack(CertificateUtil.getTrustAllCertHttpClient()))
+                //.setHttpStack(new OkHttpStack(CertificateUtil.getPinnedCertHttpClient(this, R.raw.mytruststore, "123456")))
```

Comment:
I would suggest to separate these methods and add them builder itself.
<pre>
.setHttpStack(new OkHttpStack())
.trustAllCertificates() // for all
.pinCertificates(this, R.raw.truststore, "14324") // for pinning
</pre>

I think we can also get the context already, thus we can also remove "this" for the pinning.